### DISCUSSION TITLE
what is "https://chatgpt.duti.tech/"

### DISCUSSION THREAD

**Discussion by dl942702882** [acheong08/ChatGPT#867] [General]
why the base url is "https://chatgpt.duti.tech/" ?  is it your personal proxy sever?

**Comment by acheong08** [comment]
Yes. It is to get around cloudflare

**Reply by wlc002** [reply]
HTTPSConnectionPool(host='chatgpt.duti.tech', port=443): Max retrie****ceeded with url: /api/conversation (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f072bfc8190>: Failed to establish a new connection: [Errno -2] Name or service not known'))

Is this happening because of chatgpt.duti.tech?

**Reply by acheong08** [reply]
happens with older versions of Python. use python3.10

**Reply by BorisZhang** [reply]
How can I create such a personal proxy server? Thanks

**Reply by wlc002** [reply]
the same question

**Reply by njwander** [reply]
i got the same error, but dispear after 2 hours wait, i don't think i do anything correct, but it just work.
i login the platform, i refresh the chat page, i chat with chatgpt, and then, it works!

**Reply by acheong08** [reply]
@njwander They just setup cloudflare on the endpoint I was using. I found another one
